membership_file/migrations/0001_initial.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 2.2.3 on 2019-07-16 13:17
+
+import datetime
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Member',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=63)),
+                ('tussenvoegsel', models.CharField(blank=True, max_length=15, null=True)),
```

Comment:
There actually doesn't seem to be a translation for tussenvoegsel (it's a very Dutch thing!). That being said, most members are Dutch (which'll not really change at any point in time since we are located in the Netherlands) so I'd say that having such a column here does provide some benefits.

Different cultures (and their naming conventions) do indeed provide some problems, though it's unfortunately not possible to account for every possible scenario (I learnt that when adding in the addresses of members :P). I think that we can cover most things with these fields, and that we'll 'just' have to improvise when encountering different naming conventions (E.g. filling more things into the first name field, or adding other names in the tussenvoegsel field)

Reply:
It is apparently such a Dutch thing that the [English Wikipedia has an article about it called "Tussenvoegsel"](https://en.wikipedia.org/wiki/Tussenvoegsel). Fine, the name can stay.

I don't think we can solve the different naming convention here nicely and I think splitting it up in components is the best thing we can do here, so we'll at least have clean data to work with throughout the application